jerry-debugger/jerry-client-ws.html

```diff
@@ -920,6 +927,33 @@
 
         return;
       }
+      case JERRY_DEBUGGER_OUTPUT_RESULT:
+      case JERRY_DEBUGGER_OUTPUT_RESULT_END:
+      {
+        var outputResult = concatUint8Arrays(outputResult, message);
+        var subType = outputResult[outputResult.length - 1];
+        outputResult = outputResult.slice(0, -1);
+        if (subType == JERRY_DEBUGGER_OUTPUT_OK)
+        {
+          appendLog("p_OUT: " + cesu8ToString(outputResult));
```

Comment:
For me the `p_` prefix seems strange. I would just color the output, and would not use any prefix just like gdb. Or use `out:`, `err:` kind of a short form of stdout, stderr.